### ISSUE TITLE
Enable ppc64le architecture

### ISSUE THREAD

**Issue by mtarsel** [semaphoreci/agent#243]
SemaphoreCI self hosted agent is needed on ppc64le architecture to run tests for the Calico project. The agent binary does build on ppc64le:

```
$ env GOOS=linux GOARCH=ppc64le go build -o build/agent main.go

$ file ./build/agent 
./build/agent: ELF 64-bit LSB executable, 64-bit PowerPC or cisco 7500, OpenPOWER ELF V2 ABI, version 1 (SYSV), dynamically linked, interpreter /lib64/ld64.so.2, Go BuildID=kxUqIGqsV1xnp7r7RCO6/qGCA9WRGUOCHGPXgE8fi/nDa2NKeDf_9fyOQwzZOb/S_1TkycwGy76NrmR8mBa, with debug_info, not stripped
```

There are ppc64le machines available via Oregon State University Open Source Lab (OSU OSL) that can be used for testing and development. I can assist with setup if needed. Thanks!

https://osuosl.org/services/powerdev/

**Comment by lucaspin** [comment]
@mtarsel a ppc64le binary was included in the new [v2.2.22](https://github.com/semaphoreci/agent/releases/tag/v2.2.22) version release.

**Comment by mtarsel** [comment]
awesome!! thank you very much :)
